docs/pages/includes/kubernetes-access/member-role.mdx

```diff
@@ -0,0 +1,15 @@
+```yaml
+kind: role
+version: v6
+metadata:
+  name: member
+spec:
+  allow:
+    kubernetes_groups: ["system:masters"]
+    kubernetes_labels:
+      '*': '*'
+    kubernetes_resources:
```

Comment:
Just checking - this was available in Teleport 12?

Reply:
@stevenGravy Let me know when we determine this and I'll give this PR another approval!

Reply:
Yes, that was included.